modules/connectIdSystem.js

```diff
@@ -59,7 +123,13 @@ export const connectIdSubmodule = {
       return;
     }
 
+    const storedId = getIdFromCookie() || getIdFromLocalStorage();
+    if (storedId) {
+      return {id: storedId};
```

Comment:
I have a question about what I'm doing here. I've noticed the documentation [here](https://github.com/prebid/Prebid.js/blob/master/modules/userId/index.js#L13) that states that if we return an `id` key, then the value will be stored in storage. It also makes it seem that both `id` and `callback` are optional. How can I prevent Prebid core from storing the value in storage, as we're already doing this ourselves, yet continue to allow the core logic to access the data?

Reply:
After stepping away from this, I think I can answer this question myself: We need to explicitly state in our documentation that the storage config should be omitted, then Prebid core will not store the user ID value itself?

Reply:
@jlukas79 would you mind confirming this for me?

Reply:
@jlukas79  - I'm just making another change. Spotted that we're looking for the consent data in the wrong place. Should update the PR shortly, please do not merge.

Reply:
@jlukas79  - Please review and merge whenever you are ready.